scripts/set-default-credentials.sh

```diff
@@ -9,10 +9,14 @@ if [ "${TRAVIS}" != "true" ]; then
   export TRAVIS_REPO_SLUG="firebase/firebase-tools"
 fi
 
-GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-private.json"
-if [ "${TRAVIS_REPO_SLUG}" == "firebase/firebase-tools" ]; then
-  GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-public.json"
+if [ "${LOCAL}" != "true" ]; then
```

Comment:
Might I suggest

```bash
if [ -z "$LOCAL" ]
```

https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash